internal/dispatch/caching/caching.go

```diff
@@ -6,6 +6,7 @@ import (
 	"unsafe"
 
 	"github.com/dgraph-io/ristretto"
+	"github.com/inhies/go-bytesize"
```

Comment:
You might want to check out https://github.com/dustin/go-humanize for this dependency instead. It's been around longer, has way more users, and supports more than just bytes. It might be useful in other places, too.

Reply:
Thank you! I thought I had seen a lib that did this already... changed